settings/index.html

```diff
@@ -31,7 +31,6 @@
         <div class="column is-3" v-for="device in filteredItems">
           <div class="box">
             <center>
-              <img :src="device.icon" style="height:30px;width:auto;" :title="'id = ' + device.id + ', class = ' + device.class + ', caps = ' + Object.keys(device.capabilities)"/><br/>
```

Comment:
Dit werkt niet meer?

Reply:
Niet direct nee, en aangezien dit nu op een mobile device weergegeven moet worden nam het ook wel veel ruimte in. Daarom voor nu eerst besloten om de icons weg te laten en alleen de naam te laten zien. Mogelijk kunnen we dit later nog weer inbouwen met kleinere plaatjes

Reply:
Ah natuurlijk, had ik (als desktop UI gebruiker ;) niet bij stil gestaan. Voor mobiel lijkt het me inderdaad prima om die plaatjes achterwege te laten.